src/components/Footer.tsx

```diff
@@ -0,0 +1,88 @@
+import FooterLogo from "@/assets/footerlogo.png";
+import { companyArr, resourcesArr } from "@/components/Arrays";
```

Comment:
create a folder called `static` then create a file called `companyInfo`, and etc